package.json

```diff
@@ -164,6 +165,7 @@
     "eslint-plugin-cypress": "2.12.1",
     "eslint-plugin-storybook": "0.5.13",
     "fs-extra": "10.0.0",
+    "ignore": "5.2.0",
```

Comment:
Both `ignore` and `semver` were already being used, but I referenced them directly in the schematic.